### DISCUSSION TITLE
Filter-option for individually created offers

### DISCUSSION THREAD

**Discussion by albotroz** [bisq-network/bisq#6031] [Ideas]
In order to reduce the risk for buyers and especially sellers of having problems (raid or bank-account shutdown) because they have unknowingly traded with a betrayer (e.g. the betrayer-buyer used an illegally obtained bank account for payment and therefore not his money), it would be appropriate to add filter-options when creating the offer, e.g. offers will only be accepted from users who have been trading on Bisq with their signed bank account for 30, 60, 90... days, only from certain countries or only from the country you live in, only certain banks or only the same bank and so on.
Thus, you can drastically minimize your risk of unintentionally trading with Betrayer.

I always have a queasy feeling when I open my own buy/sell request and Bisq automatically and without being asked puts me in touch with a buyer/seller who has only been on Bisq for a few days. And then in combination with a foreign bank account.

**Comment by pazza83** [comment]
Hi this is a good idea.

While this is not possible at the moment I think increasing security deposit percentage will also help to deter would be scammers. They would rather take offers where they are risking a lower percentage of btc so will be put off by higher deposit percentage offers.
